src/components/flame-graph/Settings.js

```diff
@@ -6,44 +6,70 @@
 
 import React, { PureComponent } from 'react';
```

Comment:
Let's land #826 first, which breaks out the `<Settings>` component from the call tree as a shared component. I don't think we need to duplicate this here.

Reply:
Awesome!